docs/data/routes/docs/fundamentals/cli/en.md

```diff
@@ -52,68 +110,130 @@ jss create my-jss-app nextjs --prerender SSG --fetchWith GraphQL
+|`name`|The name of the item to create or update.|`String`|
+|`displayName`|Sets the item's display name (visual name, as opposed to the actual `name` of the template).|`String`|
+|`fields`|Creates template fields. Fields can be either a plain name or `name:fieldType`. For example, `--fields Foo Bar` or `--fields Foo "Bar Bas:Rich Text"`|`Array`|
+|`icon`|Sets the icon of the item in Sitecore. For example, `People/16x16/alarmclock.png`|`String`|
+|`appName`|The name of the app. <br />**Default:** `appName` from `package.json` config.|`String`|
+|`config`|Path to `scjssconfig` file. <br />**Default:** `./scjssconfig.json`|`String`|
+|`deployUrl`|URL to the Sitecore JSS import service that accepts the package deployment.<br />**Default:** `deployUrl` from  `scjssconfig.json`|`String`|
+|`skipDeploy`|If `true`, no deployment is made, and the manifest registration code for code-first is written to the console instead.<br />**Default:** `false`|`Boolean`|
+|`acceptCertificate`|Whitelists a specific SSL certificate thumbprint, regardless of normal SSL validation. Useful for self-signed certificates.|`String`|
 
-When using Sitecore-first/connected development methodology, this command scaffolds out a new component (rendering item, datasource template, and allowed placeholder settings) in Sitecore. The developer must still create the component in the JSS app and add the component to a route with Experience Editor, but this command takes care of the rest. `jss deploy component` can be used to also modify existing components (e.g. to add fields, change field types, or set display names). On an existing component, only explicitly specified arguments apply (e.g. using `--fields` will add/modify only the fields specified, and will not delete any other fields.
+> `jss deploy template` should never be used if code-first/disconnected development is in use as it does not update the disconnected manifest, only Sitecore items.
 
-> `jss deploy component` should never be used if code-first/disconnected development is in use as it does not update the disconnected manifest, only Sitecore items.
+> `jss deploy template` has options that enable modifying how the template is scaffolded, such as specifying template fields with `--fields` or setting a display name with `--displayName`. Run `jss deploy template --help` to see all the options with descriptions.
 
-> `jss deploy component` has many options that enable modifying how the component is scaffolded, such as specifying datasource template fields with `--fields` or setting a display name with `--displayName`. Run `jss deploy component --help` to see all the options with descriptions.
+### `jss deploy`
 
-## Other scripts
+Deploys the JSS app artifacts to a Sitecore instance. Use `jss deploy --help` for subcommands.
 
-All scripts you need to run during development are defined either via the `jss` CLI command (use `--help` for options) or as package scripts in `package.json`.
+### `jss environment`
```

Comment:
Huh. I didn't know this was a thing. Does it take into account overrides to scjssconfig coming from .env file? It would probably be helpful to add this info to the doc.

Reply:
I'll test this and see.